recipes/cfflib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cfflib" %}
+{% set version = "2.0.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "748043f2064079088d57a35074cf6406c436e2537e79a500330e654aadc19ac2" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
```

Comment:
It looks like this project does not support python 3, so you should add:

```
build:
   ...
   skip: True  # [py3k]
```

Reply:
thank you

Reply:
Just to let you know the `# [py3k]` part is the selector and not a comment.